### PR TITLE
[hpc-cache] Improve help for nfs3_usage_model

### PR DESCRIPTION
**Pull request by zhoxing-ms**
Fix: #2695

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
hpc-cache

File: src/hpc-cache/azext_hpc_cache/_params.py [hunk 1]
```diff
@@ -31,6 +31,11 @@
 
 storage_target_type = CLIArgumentType(help='Name of the Storage Target.')
 
+nfs3_usage_model_type = CLIArgumentType(
+    help='Identify the primary usage model to be used for this Storage Target. '
+         'You can use command `az hpc-cache usage-model list` to see the names of usage models. '
+         'For more details about usage models, please refer to https://docs.microsoft.com/en-us/azure/hpc-cache/hpc-cache-add-storage?tabs=azure-cli#choose-a-usage-model')
```
Comment (ekpgh) [inline src/hpc-cache/azext_hpc_cache/_params.py:37]: Please remove the locale code 'en-us' from the URL - change to:
https://docs.microsoft.com/azure/hpc-cache/hpc-cache-add-storage?tabs=azure-cli#choose-a-usage-model

**Review by ekpgh — CHANGES_REQUESTED** [review]
Sorry the URL needs a change.

File: src/hpc-cache/azext_hpc_cache/_params.py (hunk 1 above)
Comment (zhoxing-ms) [reply src/hpc-cache/azext_hpc_cache/_params.py:37]: OK~ Thanks for reminding

Comment (zhoxing-ms) [inline src/hpc-cache/azext_hpc_cache/_params.py:37]:
```suggestion
         'For more details about usage models, please refer to https://docs.microsoft.com/azure/hpc-cache/hpc-cache-add-storage?tabs=azure-cli#choose-a-usage-model')

```

File: src/hpc-cache/azext_hpc_cache/_params.py [hunk 2]
```diff
@@ -31,6 +31,12 @@
 
 storage_target_type = CLIArgumentType(help='Name of the Storage Target.')
 
+nfs3_usage_model_type = CLIArgumentType(
+    help='Identify the primary usage model to be used for this Storage Target. '
+         'You can use command `az hpc-cache usage-model list` to see the names of usage models. '
+         'For more details about usage models, please refer to https://docs.microsoft.com/azure/hpc-cache/hpc-cache-add-storage?tabs=azure-cli#choose-a-usage-model')
+
+
```
Comment (zhoxing-ms) [inline src/hpc-cache/azext_hpc_cache/_params.py:40]:
```suggestion


```

**Review by ekpgh — APPROVED** [review]

**Review by Juliehzl — APPROVED** [review]

**Pull request merged**